templates/experiments/components/pagination_buttons.html

```diff
@@ -0,0 +1,32 @@
+{% if total_pages > 1 %}
+  <div class="flex {% if btn_size == 'sm' %}gap-1{% else %}justify-right{% endif %}">
```

Comment:
this should use a join to create a more cohesive set of buttons: https://v4.daisyui.com/components/join/#join